### PR TITLE
YLP-159 React 16 migration

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

**Comment by Nas84** [comment]
require these PRs : 
 -  [Viz #69](https://github.com/mdblp/viz/pull/69)
- [Tideline #50](https://github.com/mdblp/tideline/pull/50)

File: package.json [hunk 1]
```diff
@@ -97,8 +96,9 @@
     "babel-preset-react-app": "9.1.2",
     "chai": "4.2.0",
     "css-loader": "3.5.3",
+    "create-react-class": "15.6.3",
```
Comment (yvon-dblg) [inline package.json:99]: Should be in dependencies, not dev

File: package.json [hunk 2]
```diff
@@ -33,41 +33,37 @@
...
-    "body-parser": "1.19.0",
-    "compression": "1.7.4",
-    "express": "4.17.1",
-    "helmet": "3.22.0",
-    "morgan": "1.10.0",
-    "@tidepool/viz": "github:mdblp/viz#dblp.0.15.1",
-    "tideline": "github:mdblp/tideline#dblp.1.12.1",
+    "@hot-loader/react-dom": "^16.11.0",
+    "@tidepool/viz": "github:mdblp/viz#react-16-migration",
+    "@types/react": "^16.9.49",
```
Comment (yvon-dblg) [inline package.json:38]: Move it to devDependencies

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
Looks good
- package-lock.json need to be regenerated
- Missing changelog

**Review by yvon-dblg — APPROVED** [review]
LGTM

**Pull request merged**